agrirouter-sdk-dotnet-standard-test/Service/Messaging/Http/SendChunkedMessageTest.cs

```diff
@@ -106,29 +111,54 @@ public void GivenValidMessageContentWhenSendingMessageToSingleRecipientThenTheMe
             // 2. Set routes within the UI - this is done once, not each time.
             // Done manually, not API interaction necessary.
 
-            // 3. Send message from sender to recipient.
+            // 3. Add message header and message payloads.
+            var headerParameters = new MessageHeaderParameters
+            {
+                Metadata = new Metadata
+                {
+                    FileName = "my_personal_filename.png"
+                },
+                Recipients = new List<string> { Recipient.SensorAlternateId },
+                TechnicalMessageType = TechnicalMessageTypes.ImgPng
+            };
+            var payloadParameters = new MessagePayloadParameters
+            {
+                Value = ByteString.CopyFromUtf8(DataProvider.ReadRawLargeContentThatNeedsToBeChunked()),
+                TypeUrl = TechnicalMessageTypes.Empty
+            };
+
+            // 4. Chunk message content before sending it.
+            var messageParameterTuples =
+                EncodeMessageService.EncodeAndChunk(headerParameters, payloadParameters, Sender);
```

Comment:
The name `EncodeAndChunk` could be misleading, because it makes using `Encode` look unnecessary. Should we rename it?

Reply:
The method does not only chunk. Since each chunk of the raw data has to be `base64` encoded, the method encodes the raw data and prepares it for sending over the AR.

![image](https://user-images.githubusercontent.com/1309344/155505746-f380236a-bd0a-462f-93a7-bdf912114e39.png)

Therefore, the `EncodeAndChunk` is right in case the raw data has to be `base64` encoded.

Reply:
`Base64EncodeAndChunk`, then? :) It is hard for the user of the SDK to differentiate between Base64 and Protobuf encoding. We do have lots of encoding layers :)

Reply:
Fine for me, to have the last word I took `ChunkAndBase64EncodeEachChunk`. Fine for you?

Reply:
Totally, you won ;)